client/state/selectors/is-calypsoify-gutenberg-enabled.js

```diff
@@ -39,6 +41,7 @@ export const isCalypsoifyGutenbergEnabled = ( state, siteId ) => {
 			isJetpackMinimumVersion( state, siteId, '7.3-alpha' ) &&
 			( 'http:' === window.location.protocol || isHttps( getSiteAdminUrl( state, siteId ) ) )
 		) {
+			waitForData( getFrameNonce( siteId ) );
```

Comment:
Let's move this to the component that needs the nonce. It doesn't look like we use the selector in this logic.

Reply:
Oh looks like this is already being called in calypsoify iframe, so all changes on this file can be removed.